Wireframe/index.html

```diff
@@ -4,10 +4,74 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
+      <header>
+        <h1>Introduction</h1>
+        <p> Git, What is a branch in Git? and Why do developers need Git?</p>
+      </header>
+
+    <main>
+        <section class="first__paragraph">
+          <article class="layout"></article>
+        <article class="art__content">
+            <h1 class= "first__title">What is Git?</h1>
+            <p class="first__summary">Git is a version control system that helps you 
+              track changes in your files and collaborate with others on software 
+              development projects. It allows you to save 
+              different versions of your work and manage them efficiently.
+          </p>
+    
```

Comment:
Great job on using semantic HTML correctly! It's fantastic to see that you're following best practices for structuring your web pages. Keep up the good work!

To further enhance the readability of your code, I recommend using a code formatter, such as `Prettier`, for consistent and standardised indentations. It ensures that your code is consistently formatted, making it easier to read and understand.